README.rst

```diff
@@ -450,12 +450,17 @@ Authors
 
 Lightning is the work of `many contributors <https://github.com/PennyLaneAI/pennylane-lightning/graphs/contributors>`_.
 
-If you are doing research using PennyLane and Lightning, please cite `our paper <https://arxiv.org/abs/1811.04968>`_:
+If you are using PennyLane and Lightning for research, please cite `[1] <https://arxiv.org/abs/1811.04968>`_ and `[2] <https://arxiv.org/pdf/2403.02512>`_:
```

Comment:
:partying_face:

Reply:
@mlxd I recommend copying the `bibtex` here directly, I find it does make it more likely people will copy the citation (since it is easier!)

Reply:
E.g., see here for the same change I made to Catalyst: https://github.com/PennyLaneAI/catalyst/pull/918/files

In that PR:

- I added a "If you are doing research using Catalyst, please cite our paper: <bibtex codeblock>" to the readme
- made sure the same also appeared on the main docs page
- (optional) added a `cff` file so that GitHub gives you a nice copy citation button 
![image](https://github.com/user-attachments/assets/69473e40-dd51-4a1d-9004-3924f468070d)

Reply:
Good call. Will do this.

Reply:
Done.